resolwe/flow/models/entity.py

```diff
@@ -58,27 +59,59 @@ class Meta(BaseCollection.Meta):
             ("owner_relation", "Is owner of the relation"),
         )
 
+    UNIT_HOUR = 'hr'
+    UNIT_CHOICES = (
```

Comment:
@darlingt What kind-of units do you expect in time-series? Hour, seconds, days? We would like to code to be future-ready so we are thinking ahead here.

Reply:
seconds (s), minutes (min), hours (h), days (d), weeks (wk).

Weeks is not a standard ISO unit of time so I've used a common abbreviation. All others are ISO units of time and use the ISO abbreviation. I added in weeks because I think some users may use this field for age of their mice.